GameProject/Assets/OpenWorld Assets/Scripts/ThirdPersonCameraController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ThirdPersonCameraController : MonoBehaviour
```

Comment:
This class is currently in the `global::` namespace. Please provide a proper namespace.